content/developer/howtos/rdtraining/01_architecture.rst

```diff
@@ -124,6 +129,9 @@ to make sure we start on the same page during the development of our new applica
 .. _multitier architecture:
     https://en.wikipedia.org/wiki/Multitier_architecture
 
+.. _OWL framework:
+    https://odoo.github.io/owl/
+
```

Comment:
The code examples in the training are migrated to owl as well? :clap: 
(If it is, I believe it's the first time where the training is more up to date than Odoo's code :grin:)

Reply:
Maybe remove the link to owl for the moment then?  I'm not sure confusing newbies with a big repository of a js framework will help them during their first week :thinking:.

You can always add it when the training really contains js exercises :).

Reply:
Edit: Forget what I said, It can stay in the global introduction of Odoo architecture, it's a good place to introduce the framework (didn't look much into the page content before my comment, my bad).

Reply:
Ah, well it's just a mention so they know OWL exists and to keep in mind that not all of the existing JS code is good to reference for their future coding needs. At least I hope the `This will be discussed further in advanced
  topics.` line makes it obvious that the link is only there for reference. If you think it's not clear enough I can remove the link, there will just be no links to it in any of the training documents...